tests/unit/test_mysqlsh_helpers.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+import unittest
+from unittest.mock import call, patch
+
+from ops.pebble import ExecError
+
+from mysqlsh_helpers import (
+    MySQL,
+    MySQLAddInstanceToClusterError,
+    MySQLConfigureInstanceError,
+    MySQLConfigureMySQLUsersError,
+    MySQLCreateClusterError,
+)
+
+
+class TestMySQL(unittest.TestCase):
+    def setUp(self):
+        self.mysql = MySQL(
+            "127.0.0.1",
+            "test_cluster",
+            "password",
+            "serverconfig",
+            "serverconfigpassword",
+            "clusteradmin",
+            "clusteradminpassword",
+            None,
+        )
+
+    @patch("mysqlsh_helpers.MySQL._run_mysqlcli_script")
+    def test_configure_mysql_users(self, _run_mysqlcli_script):
+        """Test failed to configuring the MySQL users."""
+        _run_mysqlcli_script.return_value = b""
```

Comment:
you can set the return value in the patch line:
```
@patch("mysqlsh_helpers.MySQL._run_mysqlcli_script", return_value="")
```

Reply:
removed it (e41aa7f) since it's not needed